backend/backend-python/model/gptoss.py

```
@@ -493,37 +326,213 @@ def __init__(self, config: GPTOSSArch, layer_idx: int):
+        beta = torch.exp(page_max_score - new_max_score)
+
+        sum_exp = sum_exp * alpha + page_sum_exp * beta
+        sum_val = sum_val * alpha.unsqueeze(-1) + page_sum_val * beta.unsqueeze(-1)
+        max_score = new_max_score
+
+        return sum_val, sum_exp, max_score
+
+    def _paged_attention(
+        self,
+        queries: torch.Tensor,
+        position_ids: torch.IntTensor,
```

Ruff (0.13.1)

Unused method argument: `position_ids`

(ARG002)